app/qml/main.qml

```diff
@@ -737,42 +741,36 @@ ApplicationWindow {
         }
     }
 
-    FeaturePanel {
-        id: featurePanel
+    FormController {
+        id: formController
+
         height: window.height
```

Comment:
Is it a controller or page as the same time? (it has size and onClose signal)

Reply:
It is invisible `Item` with height and width so it can forward it to forms